packages/cursorless-engine/package.json

```diff
@@ -27,6 +27,7 @@
     "@types/mocha": "^8.0.4",
     "@types/sbd": "^1.0.3",
     "@types/sinon": "^10.0.2",
+    "fast-check": "3.12.0",
```

Comment:
Curious why fast-check is being pinned?

Reply:
Because that's what `pnpm install` did. 🤷🏽 Happy to change to anything you prefer.

Reply:
Huh yeah we should look into that. I think that has to do with a setting I set set so that pnpm would install local dependencies using `*`

I also think once we activate renovate bot we may want to pin anyway, tho there's obv a discussion to be had there